api/migrations/20241023142732-add-cohort-status.js

```diff
@@ -0,0 +1,14 @@
+const { COHORT_STATUS } = require("snu-lib");
+const { CohortModel } = require("../src/models");
+
+module.exports = {
+  async up() {
+    const archivedCohortNames = ["2019", "2020", "2021"];
+    await CohortModel.updateMany({ name: { $in: archivedCohortNames } }, { $set: { status: COHORT_STATUS.ARCHIVED } });
+    await CohortModel.updateMany({ name: { $nin: archivedCohortNames } }, { $set: { status: COHORT_STATUS.PUBLISHED } });
+  },
```

Comment:
dans les migrations on aime bien faire un for classique avec un save dedans. et un fromUser dans le save histoire de garder la traça. Mais c'est pas bien grave

Reply:
Ok.